kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/MavenRepositoryConfiguration.java

```diff
@@ -156,7 +158,15 @@ private boolean isProfileActive( Profile profile ) {
                                                      .build() );
         }
         if (settings.getActiveProxy() != null) {
-            remoteBuilder.setProxy(getActiveAetherProxyFromSettings(settings));
+            if (null == settings.getActiveProxy().getNonProxyHosts()) {
+                remoteBuilder.setProxy(getActiveAetherProxyFromSettings(settings));
+            } else {
+                Pattern p = Pattern.compile(settings.getActiveProxy().getNonProxyHosts());
```

Comment:
could `settings.getActiveProxy()` be assigned to a local variable before `(settings.getActiveProxy() != null)` to avoid repetition?

Reply:
done